shared/build.gradle.kts

```diff
@@ -136,6 +136,8 @@ kotlin {
     }
 
     iosMain.dependencies { implementation(libs.ktor.client.darwin) }
+
+    compilerOptions { optIn.add("kotlin.time.ExperimentalTime") }
```

Comment:
How does this differ from the `languageSettings.optIn` we have at the top? or can this be moved there as well? 🤔

Reply:
It works in a similar way. I was following the convention used at https://github.com/Kotlin/kotlinx-datetime/blob/5a2a6e17fc357f4c8e09cfc1bdb668720cf38bfa/core/build.gradle.kts#L225-L227 

Since I've used this across other modules, I wanted to keep the same here as well. I can change it to the use `languageSettings.optIn`